pkgs/by-name/rs/rsop/package.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitea,
+  pkg-config,
+  pcsclite,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rsop";
+  version = "0.3.9";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "heiko";
+    repo = pname;
+    rev = "rsop/v${version}";
+    hash = "sha256-K69vyZFaVvZj4yLaV/zQYoItvcTDuFR4mdmMcfl1UDA=";
+  };
+
+  cargoHash = "sha256-DJVgnfPpXf8hGX6Dv6W8GzqspMEFZHc2/Fkn1MZRXd0=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ pcsclite ];
+
+  passthru = {
+    updateScript = nix-update-script { };
+  };
+
+  meta = with lib; {
+    homepage = "https://codeberg.org/heiko/rsop";
+    description = "Stateless OpenPGP (SOP) based on rpgp";
+    license = with licenses; [
+      mit
+      apsl20
+      cc0
+    ];
+    maintainers = with lib.maintainers; [ nikstur ];
+    mainProgram = "rsop";
+  };
```

Nitpick, but would be nice as well
```suggestion
  meta = {
    homepage = "https://codeberg.org/heiko/rsop";
    description = "Stateless OpenPGP (SOP) based on rpgp";
    license = with lib.licenses; [
      mit
      apsl20
      cc0
    ];
    maintainers = with lib.maintainers; [ nikstur ];
    mainProgram = "rsop";
  };
```